src/main/java/org/c4sg/service/impl/OrganizationServiceImpl.java

```diff
@@ -88,4 +106,21 @@ public String getLogoUploadPath(Integer organizationId) {
         return LOGO_UPLOAD.getValue() + File.separator + organizationId + IMAGE.getValue();
     }
 
+	@Override
+	public List<OrganizationDTO> findByUser(Integer userId) {
+		User user = userDAO.findById(userId);
+        requireNonNull(user, "Invalid User Id");
+		List<UserOrganization> userOrganizations = userOrganizationDAO.findByUserId(userId);
+		Collections.sort(userOrganizations, (p1, p2) -> {
+			int result = 0;
+			result = p1.getStatus().compareTo(p2.getStatus());
+			return result*-1;
```

Comment:
You got to ask whoever implemented for UserProject as this was just a clone and since it passed review my assumption is this worked. The only thing I could think of is perhaps something about the order.